src/CLI/Options.cs

```diff
@@ -32,6 +32,9 @@ public class Options : IOptions
         [Option(Required = false, HelpText = "CustomUia", ResourceType = typeof(Resources.OptionsHelpText))]
         public string CustomUia { get; set; }
 
+        [Option(Required = false, HelpText = "EnableMultipleScanRoots", ResourceType = typeof(Resources.OptionsHelpText))]
+        public bool EnableMultipleScanRoots { get; set; }
```

Comment:
The name here suggests an action, not a state. Maybe `AreMultipleScanRootsEnabled` or something similar?